Get-NetView.psm1

```diff
@@ -2910,7 +2934,7 @@ function Get-NetView {
             Start-Thread ${function:CounterDetail} -Params @{OutDir=$workDir}
         }
 
```

Comment:
Given these registry commands are very expensive in terms of time, please explore whether implementing via Start-Thread allows for more concurrency vs overloading Environment().

I suggested stating with Environment(), simply because it was simpler.  But upon observing the large time execution delta, which will be significantly larger on Server vs Client.  It's best we experiment with Start-Thread.